pom.xml

```diff
@@ -25,7 +25,7 @@ SOFTWARE.
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <properties>
-    <rxjava.version>3.0.0-RC6</rxjava.version>
+    <vertx.version>4.0.0-SNAPSHOT</vertx.version>
```

Comment:
@Sammers21 let's use any constant snapshot version, e.g. `4.0.0-20200212.144500-295`

Reply:
This would not work because Vert.x is not a single artifact: we are using a few of them: `core`, `rx-java-2`, `parent`. All of them have different constants.

Reply:
@Sammers21 then we can use constant snapshot version only for `vertx-core` explicitly (until any stable version will be published). The reasons why I don't like `4.0.0-SNAPSHOT` is: 1) we'll have different dependency code in local builds/tests; 2) dependency will be updated implicitly on each new `artipie/asto` release.

Reply:
@g4s8 , done